ogre2/src/Ogre2DynamicRenderable.cc

```diff
@@ -356,6 +359,11 @@ void Ogre2DynamicRenderable::UpdateBuffer()
       this->dataPtr->ogreItem->setCastShadows(
           this->dataPtr->material->CastShadows());
     }
+    else if (lowLevelMat)
+    {
+      this->dataPtr->ogreItem->getSubItem(0)->setMaterial(lowLevelMat);
+      this->dataPtr->ogreItem->setCastShadows(castShadows);
+    }
```

Comment:
Is this needed? It looks like you're setting the material and castShadows to whatever was already stored here based on the `getCastShadows` and `getMaterial` calls above. So, I'm trying to understand if calling `setMaterial` and `setCastShadows` actually changes anything for `this->dataPtr->ogreItem`.

Reply:
the [ogreItem->_initialise](https://github.com/ignitionrobotics/ign-rendering/blob/0d43d873ecabcf401153cab11be2d20f8f6dc74f/ogre2/src/Ogre2DynamicRenderable.cc#L351) call a few lines above resets those values so I had to set them again. I added a comment to explain this. 0d43d87